methods/ga4gh_0.6.0/hhbeacon/app/server/beacon/BeaconServer.scala

```diff
@@ -0,0 +1,20 @@
+package server.beacon
+
+import server.GA4GHVersions
+
+object BeaconServer extends App {
+
+  def start(version:String) = {
+    println(s"Beacon $version Server starting...")
+    GA4GHVersions.named(version) match {
+      case GA4GHVersions.v0_5_1 => BeaconV0_5_1.start
```

Comment:
so this binding will be explicit from the build... or not if the packages changes ...